L1Trigger/L1CaloTrigger/plugins/L1CaloJetProducer.cc

```diff
@@ -546,9 +546,9 @@ L1CaloJetProducer::L1CaloJetProducer(const edm::ParameterSet& iConfig) :
 
 
 
-    isoTauBarrel.SetParameter( 0, 0.25 );
```

Comment:
@vshang 
I will merge this.  Please make a follow-up PR making these values configurable.